src/lib/Shapes.ts

```diff
@@ -26,146 +27,152 @@ import { TypedArrayValidator } from '../validators/TypedArrayValidator';
 
 export class Shapes {
-	public get string() {
-		return new StringValidator();
+	public string(options?: ValidatorOptions) {
+		return new StringValidator(options);
 	}
 
-	public get number() {
-		return new NumberValidator();
+	public number(options?: ValidatorOptions) {
+		return new NumberValidator(options);
 	}
 
-	public get bigint() {
-		return new BigIntValidator();
+	public bigint(options?: ValidatorOptions) {
+		return new BigIntValidator(options);
 	}
 
-	public get boolean() {
-		return new BooleanValidator();
+	public boolean(options?: ValidatorOptions) {
+		return new BooleanValidator(options);
 	}
 
-	public get date() {
-		return new DateValidator();
+	public date(options?: ValidatorOptions) {
+		return new DateValidator(options);
 	}
 
-	public object<T extends object>(shape: MappedObjectValidator<T>) {
-		return new ObjectValidator<T>(shape);
+	public object<T extends object>(shape: MappedObjectValidator<T>, options?: ValidatorOptions) {
+		return new ObjectValidator<T>(shape, ObjectValidatorStrategy.Ignore, options);
 	}
 
-	public get undefined() {
-		return this.literal(undefined);
+	public undefined(options?: ValidatorOptions) {
+		return this.literal(undefined, { equalsOptions: options });
 	}
 
-	public get null() {
-		return this.literal(null);
+	public null(options?: ValidatorOptions) {
+		return this.literal(null, { equalsOptions: options });
 	}
 
-	public get nullish() {
-		return new NullishValidator();
+	public nullish(options?: ValidatorOptions) {
+		return new NullishValidator(options);
 	}
 
-	public get any() {
-		return new PassthroughValidator<any>();
+	public any(options?: ValidatorOptions) {
+		return new PassthroughValidator<any>(options);
 	}
 
-	public get unknown() {
-		return new PassthroughValidator<unknown>();
+	public unknown(options?: ValidatorOptions) {
+		return new PassthroughValidator<unknown>(options);
 	}
 
-	public get never() {
-		return new NeverValidator();
+	public never(options?: ValidatorOptions) {
+		return new NeverValidator(options);
 	}
 
-	public enum<T>(...values: readonly T[]) {
-		return this.union(...values.map((value) => this.literal(value)));
+	public enum<T>(values: readonly T[], options?: ValidatorOptions) {
+		return this.union(
+			values.map((value) => this.literal(value, { equalsOptions: options })),
+			options
+		);
 	}
 
-	public nativeEnum<T extends NativeEnumLike>(enumShape: T): NativeEnumValidator<T> {
-		return new NativeEnumValidator(enumShape);
+	public nativeEnum<T extends NativeEnumLike>(enumShape: T, options?: ValidatorOptions): NativeEnumValidator<T> {
+		return new NativeEnumValidator(enumShape, options);
 	}
 
-	public literal<T>(value: T): BaseValidator<T> {
-		if (value instanceof Date) return this.date.equal(value) as unknown as BaseValidator<T>;
-		return new LiteralValidator(value);
+	public literal<T>(value: T, options?: { dateOptions?: ValidatorOptions; equalsOptions?: ValidatorOptions }): BaseValidator<T> {
```

Comment:
Hmm, is there any reason for this split in options? 👀 
Not against it, but would love to see train of thought